console/console-init/ui/src/components/ProjectHeaderCard/ProjectCard.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+import {
+  Card,
+  CardBody,
+  TextContent,
+  Flex,
+  FlexItem,
+  Divider
+} from "@patternfly/react-core";
+import {
+  CheckCircleIcon,
+  PendingIcon,
+  InProgressIcon,
+  OutlinedTimesCircleIcon
+} from "@patternfly/react-icons";
+import { IProjectCount } from "./ProjectHeaderCard";
+
+interface IProjectCardProps {
+  count: IProjectCount;
+  label: string;
+  labelShort?: string;
+}
+const ProjectCard: React.FunctionComponent<IProjectCardProps> = ({
+  count,
+  label,
+  labelShort
+}) => {
+  return (
```

Comment:
I would like to suggest destructure to avoid wrting count evrytime. 
cons {failed,total, pending etc}=count;